app/assets/javascripts/app/router.js

```diff
@@ -182,7 +182,7 @@ app.Router = Backbone.Router.extend({
   },
 
   singlePost: function(id) {
-    this.renderPage(function() { return new app.pages.SinglePostViewer({id: id}); });
+    this.renderPage(function() { return new app.pages.SinglePostViewer({id: id, el: $("#container")}); });
```

Comment:
I was pretty sure that there is a reason for this but I also had to search for a while. Without this change the timeagos in the single post view are missing and only appear after a few seconds. That's the conflict I mentioned [here](https://github.com/diaspora/diaspora/pull/7182#issuecomment-301717784). I'll move this change to a new commit.